hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2734,4 +2734,16 @@ public BatchLoadSizingStrategy getDefaultBatchLoadSizingStrategy() {
 	public boolean isJdbcLogWarningsEnabledByDefault() {
 		return true;
 	}
+
+	/**
+	 *
+	 * Override the DatabaseMetaData#supportsNamedParameters()
+	 *
+	 * @return boolean
+	 *
+	 * @since 5.1.1
```

Comment:
My preference is to not state the version it was introduced (@since 5.1.1), since it will be very easy to overlook when backporting.